service/src/io/pedestal/test.clj

```diff
@@ -80,12 +78,11 @@
 
   InputStream
   (->servlet-input-stream [wrapped-stream]
-    (proxy [ServletInputStream]
-      []
+    (proxy [ServletInputStream] []
       (available ([] (.available wrapped-stream)))
       (read ([] (.read wrapped-stream))
         ([^bytes b] (.read wrapped-stream b))
-        ([^bytes b ^Integer off ^Integer len] (.read wrapped-stream b off len)))) ))
+        ([^bytes b ^Integer off ^Integer len] (.read wrapped-stream b off len))))))
```

Comment:
Undo change

Reply:
Is this " " intentional? My editor is messing with the formatting :/